actions/loadconfig.yaml

```diff
@@ -27,11 +27,18 @@ parameters:
         position: 3
     config_file:
         type: "string"
-        description: "configuration to load to device"
+        description: "Configuration file including full path to load to device (/x/y/z.type)"
```

Comment:
How about just "Absolute path to configuration file on local system that will be sent to the device"

Reply:
That works.